debezium-server/debezium-server-pubsub/src/test/java/io/debezium/server/pubsub/PubSubIT.java

```diff
@@ -51,24 +64,34 @@
     // The topic of this name must exist and be empty
     private static final String STREAM_NAME = "testc.inventory.customers";
     private static final String SUBSCRIPTION_NAME = "testsubs";
-
     protected static Subscriber subscriber;
     private static ProjectSubscriptionName subscriptionName = ProjectSubscriptionName.of(ServiceOptions.getDefaultProjectId(), SUBSCRIPTION_NAME);
-
+    private static TopicName topicName = TopicName.of(ServiceOptions.getDefaultProjectId(), STREAM_NAME);
     {
         Testing.Files.delete(TestConfigSource.OFFSET_STORE_PATH);
         Testing.Files.createTestingFile(PubSubTestConfigSource.OFFSET_STORE_PATH);
     }
+    
+    private static ManagedChannel channel;
+    private static TransportChannelProvider channelProvider;
+    private static CredentialsProvider credentialsProvider;
 
     @AfterAll
     static void stop() throws IOException {
         if (subscriber != null) {
             subscriber.stopAsync();
             subscriber.awaitTerminated();
 
-            try (SubscriptionAdminClient subscriptionAdminClient = SubscriptionAdminClient.create()) {
+            try (SubscriptionAdminClient subscriptionAdminClient = createSubscriptionAdminClient()) {
                 subscriptionAdminClient.deleteSubscription(subscriptionName);
             }
+            
+            try (TopicAdminClient topicAdminClient = createTopicAdminClient()){
+            	topicAdminClient.deleteTopic(topicName);
```

Comment:
Do we want to do it even for non-emulator test?

Reply:
That is my intention, yes. 
Then running the tests would be simpler. One would have to only run two shell commands. And then the code is also a bit less complex. 

* Login into your Google Cloud account using `gcloud auth application-default login` as described in the [documentation](https://cloud.google.com/sdk/gcloud/reference/auth/application-default).
* ~Create a new topic `gcloud pubsub topics create testc.inventory.customers`~
* Build the module and execute the tests `mvn clean install -DskipITs=false -am -pl debezium-server-pubsub`
* ~Remove the topic `gcloud pubsub topics delete testc.inventory.customers`~

From here: https://github.com/debezium/debezium/blob/main/debezium-server/README.md?plain=1#L19-L24


The downside would be that it is no longer obvious that debezium-server does not auto-create topics, just by looking at tests.